spec/support/features/session_helper.rb

```diff
@@ -265,7 +265,7 @@ def sign_in_live_with_piv_cac(user = user_with_piv_cac)
 
     def fill_in_code_with_last_phone_otp
       accept_rules_of_use_and_continue_if_displayed
-      fill_in :code, with: last_phone_otp
+      fill_in :code, with: last_phone_otp, visible: false
```

Comment:
Actually, this makes me wonder... what's the experience for a user who has JavaScript disabled? Do we need to make sure that the real field is visible for them?

Reply:
ill take a look and play around with it because good point on the No JS aspect.